Issue #78: implement method countOpenedForTableId

crc83 opened the pull request:
Issue #78: implement method countOpenedForTableId
method implemented and tested
http://github.com/crc83/WebCash/issues/issue/78

crc83 committed:
Issue #78: implement method countOpenedForTableId 
method implemented and tested
http://github.com/crc83/WebCash/issues/issue/78

crc83 committed:
Ignoring fallen test
Add files missed in previous commit

buildhive commented:
[Sergiy Beley » WebCash #172](https://buildhive.cloudbees.com/job/crc83/job/WebCash/172/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

crc83 commented:
Failed tests:  
testGetBottom(com.siriusif.model.DinnerTableTest): expected:<79> but was:<84>

testManyToOneTableOrders(com.siriusif.service.model.DinnerTableDaoImplTest): expected:<[admin]> but was:<[sanya]>

crc83 committed:
Issue #78: implement method countOpenedForTableId 
Tests corrected
http://github.com/crc83/WebCash/issues/issue/78

buildhive commented:
[Sergiy Beley » WebCash #173](https://buildhive.cloudbees.com/job/crc83/job/WebCash/173/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

crc83 committed:
Issue #78: implement method countOpenedForTableId 
Time to wait for UI tests increased to 15 seconds
http://github.com/crc83/WebCash/issues/issue/78

buildhive commented:
[Sergiy Beley » WebCash #174](https://buildhive.cloudbees.com/job/crc83/job/WebCash/174/) FAILURE
Looks like there's a problem with this pull request
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

crc83 committed:
Issue #78: implement method countOpenedForTableId 
Fixed intedreation tests
http://github.com/crc83/WebCash/issues/issue/78

buildhive commented:
[Sergiy Beley » WebCash #175](https://buildhive.cloudbees.com/job/crc83/job/WebCash/175/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)
